evolver/history.py

```diff
@@ -10,13 +10,41 @@ class Config(BaseConfig):
         pass
 
     @abstractmethod
-    def put(self, name, data):
+    def set(self, name, data):
         pass
 
     @abstractmethod
     def get(self, query):
         pass
 
+    @classmethod
+    def track_history(cls, name: str = None):
```

Comment:
We can always create a historyserver interface and this there. At present the two seem either a little entangled or we're missing an internal aspect to the interface to allow for the backend ``history`` to be pluggable.